README.md

```diff
@@ -1,22 +1,20 @@
 # terraform-order README
 
-This is the README for your extension "terraform-order". After writing up a brief description, we recommend including the following sections.
+![Terraform Order plugin Logo](./artifacts/images/tforder-logo.png "Terraform Order plugin")
 
-## Features
-
-Describe specific features of your extension including screenshots of your extension in action. Image paths are relative to this README file.
+TERRAFORM-ORDER is a plugin aimed make terraform code easier for the IaC with TF. It was first created to create an ordering feature to order variables in alphabetical order, but it will also offers variable tracking, replacement and other functionalities that helps developers to improve the code experience.
```

Comment:
Just gramatical fix

> TERRAFORM-ORDER is a plugin aimed to make terraform code easier for the IaC with TF. It was first designed to create an ordering feature to order variables in alphabetical order, but it will also offers variable tracking, replacement and other functionalities that helps developers to improve the code experience.

Reply:
good point, will update!